### PR TITLE
refactor: use `core.ensureFastOps()`

### PR DESCRIPTION
**Pull request by petamoriken**
related: https://github.com/denoland/deno_core/issues/223

### CODE REVIEW

**Comment by bartlomieju** [comment]
Hey @petamoriken, thanks for the PR. Can you please hold-off with changes for the time being - we literally opened a PR (https://github.com/denoland/deno_core/pull/433) that implements described mechanism from the issue.

**Comment by mmastrac** [comment]
> Hey @petamoriken, thanks for the PR. Can you please hold-off with changes for the time being - we literally opened a PR ([denoland/deno_core#433](https://github.com/denoland/deno_core/pull/433)) that implements described mechanism from the issue.

Actually, @petamoriken, @bartlomieju and I just spoke a bit offline and this will actually be a great help with the work we're doing. Let's see if we can successfully land this. It's close enough to the op import we will be landing that it will be a mechanical change to take the last step of the way.

This will help quite a bit with our work, thanks for tackling it!

File: ext/crypto/00_crypto.js [hunk 1]
```diff
@@ -7,20 +7,41 @@
...
 
 import { core, primordials } from "ext:core/mod.js";
-const ops = core.ops;
 const {
+  isArrayBuffer,
+  isTypedArray,
+  isDataView,
+} = core;
+const {
+  op_crypto_base64url_decode,
```
Comment (mmastrac) [inline ext/crypto/00_crypto.js:16]: Missing `op_crypto_base64url_encode` I think

**Review by mmastrac — APPROVED** [review]
LGTM. Huge PR!

**Pull request merged**